screed/tests/__main__.py

```diff
@@ -1,8 +1,7 @@
-import os
 import sys
 
 if __name__ == '__main__':
-    if len(sys.argv) == 1:
-        sys.argv.append(os.path.dirname(__file__))
-    import nose
-    nose.main()
+    import pytest
+#    errno = pytest.main(['-m', '"not known_failing"', '-v'])
```

Comment:
This is a quirk: on a dev installation (source cloned from github), `setup.cfg` and `pytest.ini` are available, and just calling `pytest.main()` does the right thing. But when installed from PyPI, `python -m screed.tests` tries to run all tests... I don't want to keep these flags here (because it is very easy to forget that we need to change flags on multiple places), but I also didn't find a good solution...